test/input-test.jl

```diff
@@ -8,7 +8,8 @@ end
 
 # Assert frequency constructor matches the rate constructor
 @testset "ConstantRate Constructors" begin
-    freq, dt = rand(), rand()
+    freq = rand()
+    freq == 0.01 ? dt = rand() * 0.01 : dt = rand()

```

Comment:
0.01 was just an example number. In general, if `freq` is random between 0 and 1, then `dt` is random between 0 and `freq`.

Also don't use ternary operators for assignments like that. It's hard to read. The preferred style would be
```julia
destination = condition ? trueval : falseval
```